endpoints-support.md

```diff
@@ -69,7 +69,13 @@
 | <sub>/rest/connection-templates/defaultConnectionTemplate</sub>                                 | GET                       |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |   :white_check_mark:  |   :white_check_mark:  |:white_check_mark:|   :white_check_mark:  |
 | <sub>/rest/connection-templates/{id}</sub>                                                      | GET                       |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |   :white_check_mark:  |   :white_check_mark:  |:white_check_mark:|   :white_check_mark:  |
```

Comment:
API 5000 is not added?

Reply:
I have added only endpoints for now. New Api version will be added as part of validation PR.